files/recursive_equivalents.sql

```diff
@@ -8,9 +8,9 @@ $$
         WITH RECURSIVE
                 find_equivs(n, strength, input_id, output_id) AS
                 (
-                SELECT 1, 1::DOUBLE PRECISION, $1 as input_id, $1 as output_id
+                SELECT 1, 1::DOUBLE PRECISION, $1 as input_id, $1 as output_id, 0::BIGINT as r
                 UNION
-                SELECT fe.n + 1, fe.strength * e.strength, e.input_id, e.output_id
+                SELECT fe.n + 1, fe.strength * e.strength, e.input_id, e.output_id, row_number() over (order by null) as r
```

Comment:
I'm not sure how row_number works in a recursive function. Does it apply to the entire end result or each level?

Reply:
It applies to each level. So you might get up to `cutoff * recursion_depth` rows.